tests/com/cinergix/mapper/ObjectMapperTest.java

```diff
@@ -0,0 +1,49 @@
+package com.cinergix.mapper;
+
+import java.sql.ResultSet;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import com.cinergix.db.DBServiceHelper;
+import com.cinergix.mapper.data.UserMock;
+
+public class ObjectMapperTest {
+	
+	DBServiceHelper dbHelper = DBServiceHelper.getInstance();
+	@Before
+	public void beforeFunction(){
+		
+		// To create the Test database
+		dbHelper.createTestDatabase();
+	}
+	
+	@After
+	public void afterFunction(){
+		
+		// To delete the Test database
+		dbHelper.destroyDataBase();
+	}
+	
+	@Test
+	public void mapResultSetToObjectShouldReturnEmptyListIfResultSetIsEmpty(){
+		
+		ResultSet result = dbHelper.getResultSetForQuery( "SELECT * FROM user WHERE id LIKE 'testID'" );
+		ObjectMapper<UserMock> mapper = new ObjectMapper<UserMock>();
+		List<UserMock> userList = mapper.mapResultSetToObject( result, UserMock.class );
+		
+		assertNotNull( userList );
+		assertEquals( "mapResultSetToObjectShouldReturnEmptyListIfResultSetIsEmpty - Expected 0 actual " + userList.size(), 0, userList.size() );
```

Comment:
In assertion strings we can make the text more readable by making them as follows
```
assertEquals( "mapResultSetToObject should return empty list if result set is empty - Expected 0 actual " + userList.size(), 0, userList.size() );
```
If this is a good way to do it we should follow this